data/instruct_basic_bigquery.csv

```diff
@@ -11,7 +11,7 @@ broker,bigquery,basic_left_join,"SELECT c.sbCustId, c.sbCustName FROM broker.sbC
 broker,bigquery,basic_left_join,"SELECT tk.sbTickerId, tk.sbTickerSymbol FROM broker.sbTicker AS tk LEFT JOIN broker.sbDailyPrice AS dp ON tk.sbTickerId = dp.sbDpTickerId WHERE dp.sbDpTickerId IS NULL;",Return the ticker ID and symbol of tickers that do not have any daily price records.
 car_dealership,bigquery,basic_join_date_group_order_limit,"SELECT c.first_name, c.last_name, COUNT(s.id) AS total_sales, SUM(s.sale_price) AS total_revenue FROM car_dealership.sales AS s JOIN car_dealership.salespersons AS c ON s.salesperson_id = c.id WHERE s.sale_date >= CURRENT_DATE - INTERVAL '3' MONTH GROUP BY c.first_name, c.last_name ORDER BY total_revenue DESC NULLS FIRST LIMIT 3;","Who were the top 3 sales representatives by total revenue in the past 3 months, inclusive of today's date? Return their first name, last name, total number of sales and total revenue. Note that revenue refers to the sum of sale_price in the sales table."
 car_dealership,bigquery,basic_join_date_group_order_limit,"SELECT sp.first_name, sp.last_name, COUNT(s.id) AS total_sales, SUM(s.sale_price) AS total_revenue FROM car_dealership.sales AS s JOIN car_dealership.salespersons AS sp ON s.salesperson_id = sp.id WHERE s.sale_date >= CURRENT_DATE - INTERVAL '30' DAY GROUP BY sp.first_name, sp.last_name, sp.id ORDER BY total_sales DESC NULLS FIRST LIMIT 5;","Return the top 5 salespersons by number of sales in the past 30 days? Return their first and last name, total sales count and total revenue amount."
-car_dealership,bigquery,basic_join_group_order_limit,"SELECT c.state, COUNT(DISTINCT s.customer_id) AS unique_customers, SUM(s.sale_price) AS total_revenue FROM car_dealership.sales AS s JOIN car_dealership.customers AS c ON s.customer_id = c.id GROUP BY c.state ORDER BY total_revenue DESC NULLS FIRST LIMIT 5;","Return the top 5 states by total revenue, showing the number of unique customers and total revenue for each state."
+car_dealership,bigquery,basic_join_group_order_limit,"SELECT c.state, COUNT(DISTINCT s.customer_id) AS unique_customers, SUM(s.sale_price) AS total_revenue FROM car_dealership.sales AS s JOIN car_dealership.customers AS c ON s.customer_id = c.id GROUP BY c.state ORDER BY total_revenue DESC NULLS FIRST LIMIT 5;","Return the top 5 states by total revenue, showing the number of unique customers and total revenue (based on sale price) for each state."
```

Comment:
Otherwise, the model was calculating revenue based on the payment_received column which isn't completely wrong.